makefiles/default_modules.deps.mk

```diff
@@ -0,0 +1,15 @@
+# This files contains dependencies for default modules. They are parsed at the
+# end of the dependency loop. They MAY inlcude new modules, but this modules
+# MUST NOT have dependencies themselfs.
+
+ifneq (,$(filter auto_init_ztimer,$(USEMODULE)))
+  USEMODULE += ztimer_init
+endif
+
+ifneq (,$(filter auto_init_gnrc_netif,$(USEMODULE)))
```

Comment:
`auto_init_gnrc_netif` is not a DEFAULT_MODULE

Reply:
Ahh, I thought it was because of the `auto_init_` prefix..., I'll remove it from here then..